egs2/polyphone_swiss_french/asr1/run.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+# Set bash to 'debug' mode, it will exit on :
+# -e 'error', -u 'undefined variable', -o ... 'error in pipeline', -x 'print commands',
+set -e
+set -u
+set -o pipefail
+
+train_set="valid_train"
+valid_set="valid_dev"
+test_sets="valid_test"
```

Comment:
```suggestion
test_sets="valid_dev valid_test"
```
`test_sets` do not include the validation set (the validation set is only used during training). If you want to have ASR results in the validation set, you can specify like that.

Reply:
Ah ok I did not check this closely enough and copied the content from egs1.  Yes I can upload the egs2 model.

Reply:
Thanks! I copied the egs1 content without checking this change closely enough.  Yes, I can upload the egs2 model.

Reply:
I've added the egs2 reference model, as well as updated the results with the dev evaluation.  Not clear to me what the codecov issue is as this update makes no material changes to any part of the espnet core, and only only makes your recommended commit suggestion to `run.sh`, adding `valid_dev`.